Dist/packages.nix

```diff
@@ -158,7 +158,7 @@
 			''(
 				cd Assets/dll;
 				rm -f "lua54.dll" \
-					"mupen64plus-audio-bkm.dll" "mupen64plus-input-bkm.dll" "mupen64plus-rsp-cxd4-sse2.dll" "mupen64plus-rsp-hle.dll" "mupen64plus-video-angrylion-rdp.dll" "mupen64plus-video-glide64.dll" "mupen64plus-video-glide64mk2.dll" "mupen64plus-video-GLideN64.dll" "mupen64plus-video-rice.dll" "mupen64plus.dll" "octoshock.dll" \
+					"mupen64plus-audio-bkm.dll" "mupen64plus-input-bkm.dll" "mupen64plus-rsp-cxd4-sse2.dll" "mupen64plus-rsp-hle.dll" "mupen64plus-rsp-parallel.dll" "mupen64plus-video-angrylion-plus.dll" "mupen64plus-video-GLideN64.dll" "mupen64plus-video-parallel.dll" "mupen64plus.dll" "octoshock.dll" \
```

Comment:
This command is only executed when building old versions:
https://github.com/TASEmulators/BizHawk/blob/2ed8db5315d461c10164c54390d29f21e4195e10/Dist/historical.nix#L234